backend/src/main/java/ch/nova_omnia/pm4/api/AbstractCrudRestController.java

```diff
@@ -0,0 +1,46 @@
+package ch.nova_omnia.pm4.api;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import ch.nova_omnia.pm4.mapper.GenericMapper;
+import ch.nova_omnia.pm4.service.AbstractCrudService;
+
+// T: Entität, D: DTO, ID: Typ der ID (z.B. Long)
+public abstract class AbstractCrudRestController<T, D, ID> {
```

Comment:
I didn't expect to ever have to complain about too much abstraction, but this is unfortunately abstracted too much.

like this your are pre defining global http endpoint routes, which does not make sense. I'm also guessing this is why you needed a Mapper class. That should also be necessary, spring should almost automatically provide dto conversion, it uses jackson in the bg